stories/utility/CodeBlock.stories.js

```diff
@@ -25,26 +20,54 @@ https://api.sparkpost.com/api/v1/transmissions
     "text": "Sword of Omens, give me sight BEYOND sight"
   },
   "recipients": [{ "address": "email address here" }]
-}'`}/>
-    </Panel>
-  )))
+}'`;
 
-  .add('with numbers and height', withInfo({ propTablesExclude: [Panel] })(() => (
-    <Panel sectioned>
-    <CodeBlock numbered height={250} code={`curl -X POST
-https://api.sparkpost.com/api/v1/transmissions
--H "Authorization: api-key-here"
--H "Content-Type: application/json"
--d '{
-  "options": {
-    "sandbox": true
-  },
-  "content": {
-    "from": "from@email.com",
-    "subject": "Thundercats are GO!!!",
-    "text": "Sword of Omens, give me sight BEYOND sight"
-  },
-  "recipients": [{ "address": "email address here" }]
-}'`}/>
-    </Panel>
-  )));
+storiesOf('Utility|Code Block', module)
```

Comment:
Might want to convert this to match the style of the rest of the converted components. Button is a good example.